tests/functional/test_new_resolver_errors.py

```diff
@@ -101,32 +101,33 @@ def test_new_resolver_checks_requires_python_before_dependencies(
 ) -> None:
     incompatible_python = f"<{sys.version_info.major}.{sys.version_info.minor}"
 
-    pkg_dep = create_basic_wheel_for_package(
+    pkgdep = create_basic_wheel_for_package(
         script,
-        name="pkg-dep",
+        name="pkgdep",
         version="1",
     )
     create_basic_wheel_for_package(
         script,
-        name="pkg-root",
+        name="pkgroot",
         version="1",
-        # Refer the dependency by URL to prioritise it as much as possible,
-        # to test that Requires-Python is *still* inspected first.
-        depends=[f"pkg-dep@{pathlib.Path(pkg_dep).as_uri()}"],
+        depends=[
```

Comment:
Why remove the comment here? Is it no longer valid?

Reply:
Maybe to avoid needing to normalise in the check below? Again, if so that deserves a comment.

Reply:
I'll add the comment back although I can't attest to whether it's still valid. 

>  Again, if so that deserves a comment.

I don't understand why I should add a comment about the removal of another comment.

Reply:
I think the second comment was related to something else. I can't tell what from the way Github is showing it (I don't normally put 2 separate comments one after the other like this, so I think *something* changed here from what I originally wrote.

Sorry I can't recall the original context. Lets just ignore what I said here...

Reply:
No worries!